### PR TITLE
Hide secret information

### PR DESCRIPTION
**Pull request by cogutvalera**
PR to hide secret information

### CODE REVIEW

**Comment by troglobit** [comment]
Thank you for your contribution to editline! :-)

The PR cannot be merged directly due to conflicts.  There is also a lot of whitespace changes mixed in with the new functionality.  Would you mind redoing the PR; fix the conflicts and also split out (or remove) the whitespace as a separate commit?

**Comment by cogutvalera** [comment]
ok sure ! Thank you !

**Comment by cogutvalera** [comment]
I've pushed `force` new commit and added example

File: src/editline.c [hunk 1]
```diff
@@ -162,7 +162,10 @@ static void tty_flush(void)
 	return;
 
     if (!el_no_echo) {
-	res = write(el_outfd, Screen, ScreenCount);
+   if (rl_check_secret(rl_line_buffer))
+       res = write(el_outfd, "", 1);
+   else
+	    res = write(el_outfd, Screen, ScreenCount);
```
Comment (rofl0r) [inline src/editline.c]: inconsistent indentation...

Comment (cogutvalera) [reply src/editline.c]: Thanks ! fixed

**Comment by cogutvalera** [comment]
Thanks !

**Comment by troglobit** [comment]
No problem.  I didn't understand the use-case at first, and I assumed you'd already seen the `el_no_echo` global variable and dismissed it.

**Pull request merged**